Dispatcher/Supporting Files/Utils.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  Utils.swift
+//  Dispatcher
+//
+//  Created by Ofek Cohen on 03/05/2023.
+
+
+import UIKit
+
+func formatDate(_ date: String) -> String {
+    let dateFormatter = DateFormatter()
+    dateFormatter.dateFormat = "EEEE, MMM d, yyyy"
+    dateFormatter.locale = Locale(identifier: "en_US")
+    let fomattedDate = dateFormatter.date(from: date) ?? Date()
+    return dateFormatter.string(from: fomattedDate)
+}
+
+func loadImageFromUrl(_ imageUrl: String) -> UIImage {
+    if let url = URL(string: imageUrl) {
+        if let data = try? Data(contentsOf: url) {
+            if let image = UIImage(data: data) {
+               return image
+            }
+        }
+    }
+
+    return UIImage()
+}
+
+
+func createErrorAlert(_ msg: String) -> UIAlertController {
```

Comment:
* The naming convention would be to use "getErrorAlert" rather than "createErrorAlert"
* It would be nicer to read when calling this function if the signature was: getErrorAlert(withMessage msg: String)